### PR TITLE
New and final updates

### PR DESCRIPTION
**Pull request by majidism006**
Here are some new and final updates guys! Go team!

### CODE REVIEW

**Review by RaghavAtRuntime — DISMISSED** [review]
Great job cleaning up the code.

File: src/main/java/use_case/grid/GridInputData.java [hunk 1]
```diff
@@ -49,14 +49,7 @@ public String getLetter() {
         return letter;
     }
 
-    /**
-     * Gets the guessed word.
-     * (Currently returns an empty string, to be implemented)
```
Comment (ibrmaj) [inline src/main/java/use_case/grid/GridInputData.java:54]: Please add JavaDoc related to the method

File: src/main/java/view/InstructionsView.java [hunk 2]
```diff
@@ -172,6 +172,7 @@ public void actionPerformed(ActionEvent e) {
                 ActionEvent.ACTION_PERFORMED, null));
     }
 
+    // This creates the visual examples for the user.
     private JPanel createExample(String word, int highlightedIndex) {
```
Comment (ibrmaj) [inline src/main/java/view/InstructionsView.java:176]: this is not really java doc its more like a comment, what you had up there in SwitchToProfileView is proper JavaDoc

File: src/main/resources/easy_words [hunk 3]
```diff
@@ -235,11 +235,340 @@ VISIT
...
+PAGES
+SLEEP
+ITEMS
+FRESH
+KNOWS
+DRIVE
+RURAL
+METAL
+GUIDE
+STEEL
```
Comment (ibrmaj) [inline src/main/resources/easy_words:284]: Words are not really easy, should have used the website linked on discord for easier words, I would suggest move more of these to the hard_words file

File: src/main/java/use_case/instructions/InstructionsInteractor.java [hunk 4]
```diff
@@ -68,6 +68,9 @@ public void switchToDiscussionBoardView() {
         instructionsPresenter.switchToDiscussionBoardView();
     }
 
+    /**
+     * Switches to Profile View.
```
Comment (ibrmaj) [inline src/main/java/use_case/instructions/InstructionsInteractor.java:72]: Good JavaDoc!

**Review by ibrmaj — CHANGES_REQUESTED** [review]

File: src/main/java/use_case/instructions/InstructionsInteractor.java (hunk 4 above)
Comment (majidism006) [reply src/main/java/use_case/instructions/InstructionsInteractor.java:72]: Thank you!

**Comment by ibrmaj** [comment]
Nice refactoring, maybe a bit more cleanup would help, I have added the needed things that should be resolved!

**Review by ibrmaj — APPROVED** [review]

**Comment by ibrmaj** [comment]
Good Job!

File: src/main/java/use_case/grid/GridInputData.java [hunk 5]
```diff
@@ -50,13 +50,11 @@ public String getLetter() {
...
 
     /**
-     * Gets the guessed word.
-     * (Currently returns an empty string, to be implemented)
+     * Gets the guess placed in the grid.
      *
-     * @return the guessed word
+     * @return the letter
      */
     public String getGuess() {
```
Comment (CaaarolXu) [inline src/main/java/use_case/grid/GridInputData.java:57]: This method is duplicated by getLetter(). Please delete one of them.

Comment (CaaarolXu) [reply src/main/java/use_case/grid/GridInputData.java:57]: getLetter() is never used, so I think we can delete it.

**Review by CaaarolXu — CHANGES_REQUESTED** [review]

**Review by CaaarolXu — APPROVED** [review]

**Pull request merged**